examples/language_model/gpt-3/dygraph/run_pretrain.py

```diff
@@ -30,26 +30,20 @@
 import lr
 from paddle.distributed import fleet
 from paddle.distributed.fleet.meta_parallel import get_rng_state_tracker
+from paddle.distributed.fleet.meta_optimizers.dygraph_optimizer.dygraph_sharding_optimizer import DygraphShardingOptimizer
```

Comment:
It's too long for this import ,is it ok to update the init file to shorter the import path?